pkgs/by-name/ca/cargo-information/package.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, openssl
+, stdenv
+, darwin
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-information";
+  version = "0.4.2";
+
+  src = fetchFromGitHub {
+    owner = "hi-rustin";
+    repo = "cargo-information";
+    rev = "v${version}";
+    hash = "sha256-k481iHQ1tVi9fF5/xYR99/1/oRv1nS3WH7W55aPSyfc=";
+  };
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+    outputHashes = {
+      "cargo-test-macro-0.1.0" = "sha256-4u3Ium+WYBdyocuehDulRgUOR74JC6AUI2+A5xlnUGw=";
+    };
+  };
+
+  checkFlags = [
+    # Require network access
+    "--skip=cargo_information::specify_version_within_ws_and_match_with_lockfile::case"
+    "--skip=cargo_information::within_ws::case"
+    "--skip=cargo_information::within_ws_with_alternative_registry::case"
+    "--skip=cargo_information::within_ws_without_lockfile::case"
+  ];
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs = [
+    openssl
+  ] ++ lib.optionals stdenv.isDarwin [
+    darwin.apple_sdk.frameworks.Security
+  ];
+
```

Comment:
```suggestion
postFixup = ''
  wrapProgram $out/bin/cargo-info \
    --prefix PATH : ${lib.makeBinPath [ rustc ]}
'';
```

```
=> cargo-info info libadwaita
error: could not execute process `rustc -vV` (never executed)

Caused by:
  No such file or directory (os error 2)
```

Reply:
Thank you https://github.com/NixOS/nixpkgs/compare/62be6e81405cfd2dd47caf7074f297995b20618c..0feae8bf3ccc1945d85f6a7d688242161ed6fd7a

I managed to reproduce this by creating a pure shell with no packages and running the binary there. Do you know if there's also an easier way to do it, perhaps a flag I can pass to `nix run .#cargo-information`?

Reply:
Oops, I missed adding `makeWrapper` to my inputs. I'll push a fix

Reply:
https://github.com/NixOS/nixpkgs/compare/0feae8bf3ccc1945d85f6a7d688242161ed6fd7a..fba585953a0b8211c5ae81fd6ed777ab81d2a335